hashistack/part_2_hashistack/playbooks/hashistack.yml

```diff
@@ -11,9 +11,10 @@
       consul_domain: "consul"
       consul_recursors: ['1.1.1.1', '1.0.0.1']
       # CONSTANTS
-      consul_client_address: "0.0.0.0" 
+      consul_client_address: "0.0.0.0"
       consul_dnsmasq_enable: yes
       consul_acl_master_token_display: yes
+      vault_azurekeyvault: no
```

Comment:
What is the purpose of this?

Reply:
This sets the boolean value of "vault_azurekeyvault" explicitly to false because otherwise Ansible will stop with the following error:

```
TASK [brianshumate.vault : Listener configuration] *************************************************************************************************
fatal: [server01.consul]: FAILED! => {"changed": false, "msg": "AnsibleUndefinedVariable: 'vault_azurekeyvault' is undefined"}
```